### PR TITLE
Landingpage

### LINKED ISSUE #33: [Feature Request] Landing page required
### Is your feature request related to a problem?

Yes, it is

### Please describe the problem this request is related to. If you didn't mark the checkbox above, please skip this question.

Upon opening the localhost:4200/ as described by the readme of this repository, all I can see is a blank page consisting of a 'header' and a long red line of 'Hello World' which is unresponsive and insensitive.
Screenshot from 2022-08-01 12-36-45

### Please describe solution to you'd like to see.

In my opinion, I think it should be replaced by a landing page that welcomes newbies into the system with a signup/login form and other details about the report-card system (for example, a navigation bar which includes About, Help, Contact, etc).

### Please describe the benefit of your feature.

The page will be used to encourage visitors to the reportcard system to take action (by either signing up or getting a free trial, for example). The landing page will help in providing information about the item(in this case, the reportcard system).

### Please describe the alternatives you have considered.

I do not think there are any alternatives to a landing page for this system since the landing page is basically one of the most used web tools to attract a visitor's attention.

### PR DESCRIPTION
**Pull request by de-lia**
### Contributor Checklist

Fixes #33 

- [x] Did you add comments for every public function/method?
- [x] Did you specify the type of improvement?
- [ ] Did you request a desired merge strategy?

#### Type of Improvement

- Performance Improvement
- New feature/API

#### Backend Code Generation Impact

- It will be flexible to changes in the requirements of the software.

#### Desired Merge Strategy

-Squash

#### Release Notes

-

### Reviewer Checklist (only modified by reviewer)
- [ ] Did you add the appropriate labels?
- [ ] Did you review?
- [ ] Did you check whether all relevant Contributor checkboxes have been checked?

### CODE REVIEW

File: src/app/components/rc-welcome/rc-welcome.component.html [hunk 1]
```diff
@@ -4,8 +4,8 @@
     <div class="text-6xl text-primary font-bold mb-3">Transform the way you manage the school!</div>
     <p class="mt-0 mb-4 text-700 line-height-3">.</p>
 
-    <button [routerLink]="['/auth']" pButton pRipple label="Sign Up" icon="pi pi-user-plus" iconPos="left" type="button" class="mr-3 p-button-raised border-noround"></button>
-    <button [routerLink]="['/auth']" pButton pRipple label="Sign In" icon="pi pi-sign-in" iconPos="right" type="button" class="p-button-outlined border-noround"></button>
+    <button [routerLink]="['/auth']" pButton pRipple label="Sign Up" icon="pi pi-user-plus" iconPos="left" type="button" class="mr-3 p-button-raised border-noround">Sign Up</button>
+    <button [routerLink]="['/auth']" pButton pRipple label="Sign In" icon="pi pi-sign-in" iconPos="right" type="button" class="p-button-outlined border-noround">Sign In</button>
```
Comment (elroykanye) [inline src/app/components/rc-welcome/rc-welcome.component.html:8]: I don't think these are necessary as the text on the buttons are already there.

Comment (de-lia) [reply src/app/components/rc-welcome/rc-welcome.component.html:8]: Oh. Didn't see that. When I pulled I couldn't find labels for the buttons so I just thought I put some.

Comment (elroykanye) [reply src/app/components/rc-welcome/rc-welcome.component.html:8]: You could check for that on the UI display to confirm it's an issue first, or write me up.
Hopefully, that's how you will be doing that now. Don't fail to let me know if something else confuses you.

Comment (elroykanye) [reply src/app/components/rc-welcome/rc-welcome.component.html:8]: Please revert the commit

**Review by elroykanye — APPROVED** [review]

**Comment by elroykanye** [comment]
Hi @de-lia , thanks for your help on this. We can merge this as it is now and work on more updates in future. Hope you are doing well!

**Pull request merged**